spec/adapters/jira-spec.js

```diff
@@ -2,41 +2,91 @@ import { jsdom } from 'jsdom';
 
 import adapter from '../../src/common/adapters/jira';
 
-// const SIDEBAR = ``;
+const SIDEBAR = `<div>
+  <div class="ghx-fieldname-issuekey">
+    <a href="#">UXPL-39</a>
+  </div>
+  <div data-field-id="summary">A Random JIRA Sidebar Issue</div>
+  <div data-issue-key="UXPL-39">
+    <span class="ghx-type" title="Story"></span>
+  </div>
+</div>`;
```

Comment:
Could you please make the string formatting here consistent with the rest of the HTML snippets?

I.e. everywhere else, we do

```javascript
const SNIP = `
  <div>
    …
  </div>
`;
```

so opening and closing tags are indented the same.